src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -134,13 +134,17 @@ CustomTextLayout::CustomTextLayout(IDWriteFactory1* const factory,
         RETURN_IF_FAILED(_analyzer->AnalyzeScript(this, 0, textLength, this));
         RETURN_IF_FAILED(_analyzer->AnalyzeNumberSubstitution(this, 0, textLength, this));
 
+#ifdef EXTERNAL_BUILD
         // Perform our custom font fallback analyzer that mimics the pattern of the real analyzers.
         // Fallback routines are not available below Windows 8.1, so just skip them and let a replacement character happen.
         if (IsWindows8Point1OrGreater())

```

Comment:
It's probably generally better to just QueryInterface for something that only exists on 8.1+ :smile: